packages/apps/dev-wallet/src/modules/account/account.service.ts

```diff
@@ -112,7 +119,7 @@ export const accountDiscovery = (
         await emit('key-retrieved')(key);
         const chainResult = (await discoverAccount(
           `k:${key.publicKey}`,
-          networkId,
+          network.networkId,
           undefined,
           contract,
         )
```

Comment:
Should the variable name change to `networkUUID` for `discoverAccount`s 2nd argument?

Reply:
`discoverAccount` is imported form `client-utils` it only needs networkId,
the client is injected via the wallet so it will work with correct network incase if there are two networks with same networkId